management-interface.go

```diff
@@ -39,21 +44,82 @@ func showDiskSpace(w http.ResponseWriter) error {
 		fmt.Fprintf(w, "Cannot show disk space at this time.\n")
 		return err
 	}
-	fmt.Fprintf(w, "%s\n", out)
+	fmt.Fprintf(w, "Disk space usage is: \n\n%s\n", out)
 	return nil
 
 }
 
+// Show the available network interfaces on this machine.
+func availableInterfaces(w http.ResponseWriter) error {
+
+	interfaces, err := net.Interfaces()
+
+	if err != nil {
+		return err
+	}
+
+	fmt.Fprintf(w, "Available network interfaces on this machine:\n\n")
+	for _, i := range interfaces {
+		if i.Name != "lo" {
+
+			fmt.Fprintf(w, "Testing %v...\n", i.Name)
+			fmt.Fprintf(w, "  Sending %v packets\n", NumPacketsToSend)
+
+			// Windows
+			//commandStr := "ping -n " + strconv.Itoa(NumPacketsToSend) + " -w 15 1.1.1.1"
+			//out, _ := exec.Command("cmd", "/C", commandStr).Output()
+			//pos := strings.Index(outStr, "Received =")
+			//if pos != -1 {
+			//	numPacketsReceivedStr := string(outStr[pos+11])
+
+			// Unix
+			commandStr := "ping -I " + i.Name + " -c " + strconv.Itoa(NumPacketsToSend) + " -n -W 15 1.1.1.1"
+			out, _ := exec.Command("sh", "-c", commandStr).Output()
```

Comment:
There shouldn't be a need to invoke `sh` to run ping. You should just be able to do something like this:
```go
exec.Command("ping", "-I", i.Name, "-c", strconv.Itoa(NumPacketsToSend), "-n", "-W", "15", "1.1.1.1")
```